graphql_compiler/tests/schema_transformation_tests/test_merge_schemas.py

```diff
@@ -554,7 +554,13 @@ def test_invalid_identifiers(self):
             )
         with self.assertRaises(ValueError):
             merge_schemas(
-                OrderedDict([(42, parse(ISS.basic_schema)), ("enum", parse(ISS.enum_schema)),]), [],
+                OrderedDict(
+                    [
+                        (42, parse(ISS.basic_schema)),  # type: ignore
```

Comment:
Why the `type: ignore`? Can we resolve with a cast instead?

Reply:
Casting `42` to `str`? This test ensures that you can't pass an `int` as the identifier. Since the type hint specifies that the identifier must be a `str`, we have to ignore the mypy error in order to make sure that this raises a `ValueError`. Let me know if I'm misunderstanding you!

Reply:
Oooh I see. No worries, this is fine.